### PR TITLE
example: Add server example

### LINKED ISSUE #42: Need to implement an example of server
## Feature Request

**Is your feature request related to a problem? Please describe.**  

In order to make it easier to understand how to use the library, we should implement an actual usage example

**Describe the solution you'd like**  

We need to implement an example in the server that verifies information using the library.

**Describe alternatives you've considered**  

**Additional context**

### PR DESCRIPTION
**Pull request by HyeockJinKim**
# Summary 
Add an example to book a movie
even if the server does not store user information.

- It closes #42

## How to Review? 

- Please refer to my self-review submitted as the review.

### CODE REVIEW

File: examples/server/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+const privateKey = {
+  key: '2ef40452ec154cd38efdc8ffa52e7f513f7d2b2a77e028342bde96c369e4f77a',
+  type: KeyType.ECDSA,
+};
+
+const publicKey = {
+  key: '048da7b63430eb4db203177baf2e8699a25116561624e67a31c2bf288d54216ce3f6f9c7b81fdbb5732342475a6ee5ccab883277ddbb38fdb79ab5424d401b844a',
+  type: KeyType.ECDSA,
+};
```
Comment (HyeockJinKim) [inline examples/server/src/index.ts:27]: I think we need a function that gets the public key from the private key.

File: examples/server/src/utils.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  return address.length > 5
+}
+
+export function validateAge(age: string): boolean {
+  // Assume a verification request from a trusted authority.
+  //
+  // Initially verified by a trusted authority, and when used later,
+  // it is no longer verified by a trusted authority.
+  return true;
+}
```
Comment (HyeockJinKim) [inline examples/server/src/utils.ts:20]: Assumes verification by a trusted authority.
In the example, it simply returns true.

File: examples/server/test/example-data.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      "authType": "PRIVACY",
+      "keyType": "ECDSA",
+      "value": "3046022100d7719965b55c81d9a078520ce9de3f3da606346fcbb2f9895de2999aafe6e224022100a99df25becc64fc19397f077c0033e7dd663d8d7dc2e71655a0c84b83c3213a3"
+    }
+  },
+  "view/movie/1 Example": {
+    "name": "Les Misérables",
+    "seat": 1
+  }
+}
```
Comment (HyeockJinKim) [inline examples/server/test/example-data.json:64]: Data used in request and reply.

**Review by HyeockJinKim — COMMENTED** [review]
self review

**Review by anthonyminyungi — APPROVED** [review]
A quite simple and clear example. Cool :)

**Pull request merged**